sisl/io/siesta/out.py

```diff
@@ -249,18 +249,39 @@ def read_force(self, last=True, all=False):
         all: bool, optional
            return a list of all forces (like an MD)
            If `True` `last` is ignored
+        total: bool, optional
+            return the total forces instead of the atomic forces.
+        maxF: bool, optional
+            whether only the maximum atomic force should be returned for each step.
+            If `True`, `total` is ignored.
 
```

Comment:
Allow both `total` and `max` at the same time.

Reply:
Will do!